remix_exercises/Challenge0.sol

```diff
@@ -0,0 +1,14 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+interface Challenge0 {
+    function callMe() external;
+}
+
+contract Challenge0 {
+    event Winner(address);
+
+    function callMe() external {
+	emit Winner(msg.sender);
```

Comment:
Can't seem to make a suggestion for some reason but it looks like there isnt any indentation in the function body

Reply:
Weird, ok I can fix that.